bubs/tokenizer.py

```diff
@@ -31,7 +31,7 @@ def _html_tokenize(sentence):
 
 
 class RegexTokenizer:
-    """Fast regex-based tokenizer, wrapping around https://github.com/fnl/segtok"""
+    """Fast regex-based tokenizer, wrapping around https://github.com/fnl/segtok."""
```

Comment:
Periods are legal in URLs so editors (and I) would read this URL as `https://github.com/fnl/segtok.` Consider adding a space before the period or rephrasing the docstring to put the URL in a comment?